builtin/game/item_entity.lua

```diff
@@ -17,6 +17,23 @@ end
 local time_to_live = tonumber(core.settings:get("item_entity_ttl")) or 900
 local gravity = tonumber(core.settings:get("movement_gravity")) or 9.81
 
+local function enable_physics(object, luaentity)
+	if luaentity.physical_state == false then
+		luaentity.physical_state = true
+		object:set_properties({physical = true})
+		object:set_velocity({x=0, y=0, z=0})
+		object:set_acceleration({x=0, y=-gravity, z=0})
+	end
+end
+
+local function disable_physics(object, luaentity)
```

Comment:
Wouldn't it suffice to pass only one of these parameters? The other parameter can be derived within the function itself.